src/Http/Controllers/SettingController.php

```diff
@@ -18,18 +19,24 @@ class SettingController extends Controller
      */
     public function index() {
         $tags = Tag::all();
+        $notification_channels = Integration::where('type', 'slack')->get();
 
         return view('calendar::setting.index', [
-            'tags' => $tags
+            'tags' => $tags,
+            'slackIntegrations' => $notification_channels,
         ]);
     }
 
     /**
      * @param \Illuminate\Http\Request $request
      * @return \Illuminate\Http\RedirectResponse
      */
-    public function updateSlack(Request $request) 
+    public function updateSlack(Request $request)
     {
+        setting([
+            'kassie.calendar.slack_integration_default_channel',
+            $request->slack_integration_default_channel,
```

Comment:
should add a validation to ensure input value is valid
https://laravel.com/docs/6.x/validation#quick-writing-the-validation-logic
https://laravel.com/docs/6.x/validation#rule-exists